x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/range_editor.tsx

```diff
@@ -6,21 +6,73 @@
 
 import React, { useEffect, useState } from 'react';
 import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
 import {
   EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiCode,
+  EuiFlexGroup,
+  EuiFlexItem,
   EuiFormRow,
   EuiRange,
-  EuiFlexItem,
-  EuiFlexGroup,
-  EuiButtonIcon,
   EuiToolTip,
-  EuiIconTip,
 } from '@elastic/eui';
-import { IFieldFormat } from 'src/plugins/data/public';
+import type { IFieldFormat } from 'src/plugins/data/public';
+import { UI_SETTINGS } from '../../../../../../../../src/plugins/data/public';
 import { RangeColumnParams, UpdateParamsFnType, MODES_TYPES } from './ranges';
 import { AdvancedRangeEditor } from './advanced_editor';
 import { TYPING_DEBOUNCE_TIME, MODES, MIN_HISTOGRAM_BARS } from './constants';
 import { useDebounceWithOptions } from '../helpers';
+import { HelpPopover, HelpPopoverButton } from '../../../help_popover';
+
+const GranularityHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+          {i18n.translate('xpack.lens.indexPattern.ranges.granularityHelpText', {
+            defaultMessage: 'How does it work?',
+          })}
+        </HelpPopoverButton>
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.lens.indexPattern.ranges.granularityPopoverTitle', {
+        defaultMessage: 'How does interval granularity work?',
+      })}
+    >
+      <p>
+        {i18n.translate('xpack.lens.indexPattern.ranges.granularityPopoverBasicExplanation', {
+          defaultMessage:
+            'Interval granularity divides the field into evenly spaced intervals based on the minimum and maximum values for the field.',
+        })}
+      </p>
+
+      <p>
+        <FormattedMessage
+          id="xpack.lens.indexPattern.ranges.granularityPopoverExplanation"
+          defaultMessage='The size of the interval is chosen to be a "nice" value. It is possible for the chosen
+            interval to stay the same when changing the granularity slider if the "nice" interval is
+            the same. The minimum granularity is 1, and the maximum is 
+            {setting}. To change the maximum granularity setting, go to Advanced settings.'
```

Comment:
@MichaelMarcialis Can popovers have links?  If so, it might be nice to link to Advanced Settings.  Or, are they similar to tooltips, which don't allow links.

Reply:
Yes, popovers can include links.